Environment/NoiseLevelObserved/doc/spec.md

```diff
@@ -40,15 +40,15 @@ This entity is primarily associated with the Smart City and environment vertical
 It can be represented by an specific time instant or by an ISO8601 interval. As a workaround for
```

Comment:
According to the changes introduced in https://github.com/Fiware/dataModels/pull/246, `dateCreated` and `dateModified` above in this file (around L14) should change "Optional" to "Read-Only. Automatically generated."

Reply:
(Maybe the change is already in master and it is a matter of just updateing this PR branch with master... not sure)